### ISSUE TITLE
0.3.7-preview8 breaks UI.Image()

### ISSUE THREAD

**Issue by paulmelis** [StereoKit/StereoKit#571]
## Description

Updated my model viewer app to use 0.3.7-preview8 and now `UI.Image` and some buttons show incorrect images:

Before:

![image](https://user-images.githubusercontent.com/1408287/212654362-3b7482d1-02be-43b7-9768-c67ea3fc60d3.png)

After:

![image](https://user-images.githubusercontent.com/1408287/212654468-93cea68d-6303-43d4-bf1c-3495b27e253f.png)

## Platform / Environment

Windows

**Comment by maluoi** [comment]
Woah, yeah that doesn't exactly look right! `UI.Toggle` and `UI.Radio` all now come with an image to indicate their state and behavior, and to highlight that they're different from just a `UI.Button`. You can still create a Toggle/Radio behavior _without_ the image using `UI.Toggle(UIBtnLayout.None)`, though I don't believe I have that exact overload directly for `Radio`. `Radio` is just a Toggle with a wrapper to remove the `ref` keyword though.

The sized buttons definitely need some work though, and I'll need to look into how to do that better!

**Comment by maluoi** [comment]
Yeah, pressed color is still high on my list right now! That's #506. I'll make sure Radio gets the same overloads as Toggle here :)
